src/main/java/org/broadinstitute/dsm/analytics/GoogleAnalyticsMetrics.java

```diff
@@ -0,0 +1,29 @@
+package org.broadinstitute.dsm.analytics;
+
+public class GoogleAnalyticsMetrics {
+
+//    public static final String EVENT_CATEGORY_USER_REGISTRATION = "user-registration";
+//    public static final String EVENT_ACTION_USER_REGISTRATION = "register-user";
+//    public static final String EVENT_LABEL_USER_REGISTRATION = "registration"; we may need it for FON
+
+    public static final String EVENT_CATEGORY_USER_LOGIN = "user-login";
+    public static final String EVENT_ACTION_USER_LOGIN = "user-logged-in";
+    public static final String EVENT_LABEL_USER_LOGIN = "login"; //studyGuid appended
```

Comment:
I haven't added these events to the code yet, it wasn't in the ticket, so I wasn't sure